src/containers/Article.js

```diff
@@ -121,13 +125,11 @@ class Article extends PureComponent {
   constructor(props, context) {
     super(props, context)
     this.state = {
-      fontSize:'medium',
-      isFontSizeSet:false
+      fontLevel: props.fontLevel
```

Comment:
Do we still need `state.fontLevel` here? Or can we just get the value from props given by redux?